helm/cluster-operator/templates/configmap.yaml

```diff
@@ -15,16 +15,14 @@ data:
         address: 'http://0.0.0.0:8000'
     guest:
       cluster:
+        baseDomain: '{{ .Values.baseDomain }}'
         calico:
           subnet: '{{ .Values.cni.subnet }}'
           cidr: '{{ .Values.cni.mask }}'
         kubernetes:
           api:
-            clusterIPRange: '{{ .Values.kubernetes.api.clusterAPIRange }}'
+            clusterIPRange: '{{ .Values.kubernetes.api.clusterIPRange }}'
```

Comment:
Did you test this and did it work before?
`cluster-operator-cluster-operator-3.7.2-capi` is crashlooping now on `ginger` and I see this error:
```
panic: {"kind":"invalidConfigError","annotation":"invalid CIDR address: ","stack":[{"file":"/root/project/service/controller/key/key.go","line":32},{"file":"/root/project/service/service.go","line":122}]}
```

The reason seems to be that the value for `clusterIPRange` in the configmap is empty.

Reply:
I tested it before I fixed it, but I missed a spot, namely `helm/values.yaml`. Fixed by https://github.com/giantswarm/cluster-operator/pull/1357/commits/9f13fb7841d03486fc5cc87334fe650e81d0febe.
I should probably re-release. What would the version be  `3.7.2-capi1`? `3.7.3-capi`?

Reply:
Awesome :) 
I don't have any preference there. Just go with what sounds right to you.

Reply:
[v3.7.2-capi-1](https://github.com/giantswarm/cluster-operator/releases/tag/v3.7.2-capi-1) is now a thing.

Reply:
Nice, I'll replace it in the release.

Reply:
Eh, seems our CI does not like what I did one bit: https://app.circleci.com/pipelines/github/giantswarm/cluster-operator/2517/workflows/b0eed0ef-8966-46ab-9921-771a1f101105/jobs/12571
I'll have to create another release and remember about `pkg/project/project.go` this time :sob: